hw/dv/tools/vcs/cover.cfg

```diff
@@ -19,3 +19,14 @@ begin tgl
   +module prim_prince
   +module prim_lfsr
 end
+
+begin assert
+  // These three assertions in prim_lc_sync check when `lc_ctrl_pkg::lc_tx_e` input is neither `On`
+  // or `Off`, it is interrupted to the correct `On` or `Off` after one clock cycle. This behavior
+  // is implemented outside of IP level design thus these assertions are not covered in IP level
+  // testbenchs.
+  // TODO: check in top-level or FPV.
+  -assert CheckTransients0_A
```

Comment:
Would you please check the VCS coverage userguide to see if there is a way to exclude an assertion from coverage in a specific module? 

Otherwise, some other RTL module with the same label might be excluded as well. This case may or may not exist, but better to not take chance.

Reply:
Thanks Sri for the suggestion, I have not found anything in the manual. Let me see if the AE has any suggestions.

Reply:
Does `-assert *prim_lc_sync*.CheckTransients*A` work?

Reply:
unfortunately no.. because each IP has its own naming when instantiate this `prim_lc_sync` module.
For example：
otp_ctrl called it: `u_prim_lc_sync_dft_en.CheckTransients_A`,
clkmgr called it:  `u_io_dvi4_div_scanmode_sync.CheckTransients_A`,

Reply:
In that case, can we make these assertion labels harder to accidentally replicate in some other module?
Perhaps rename them to `PrimLcSyncCheckTransients*_A`?

Reply:
Sure, I think that would work!